server/src/main/scala/org/broadinstitute/workbench/hamm/server/HammGrpcImp.scala

```diff
@@ -0,0 +1,51 @@
+package org.broadinstitute.workbench.hamm
+package server
+
+import java.time.Instant
+
+import cats.effect.Sync
+import cats.implicits._
+import io.chrisdavenport.log4cats.Logger
+import io.grpc.Metadata
+import org.broadinstitute.workbench.hamm.CostCalculator
+import org.broadinstitute.workbench.hamm.pricing.JsonCodec._
+import org.broadinstitute.workbench.hamm.pricing.{GcpPricing, PriceList}
+import org.broadinstitute.workbench.hamm.protos.hamm._
+
+class HammGrpcImp[F[_]: Sync: Logger](pricing: GcpPricing[F]) extends HammFs2Grpc[F] {
```

Comment:
What's an `Imp`?

Reply:
Implementation...? I'm not sure - we can name it something better.